Cargo.toml

```
@@ -414,9 +414,10 @@ vrl.workspace = true
 wiremock = "0.6.2"
 zstd = { version = "0.13.0", default-features = false }
 
-tokio-rustls = "0.23"
-rustls = "0.20"
+tokio-rustls = "0.26"
+rustls = "0.23"
```

```suggestion
rustls = { version = "0.23", features = ["fips"] }
```

Let's wait for this fix: https://github.com/aws/aws-lc-rs/issues/587#issuecomment-2477299823